packages/puppeteer-core/src/bidi/HTTPRequest.ts

```diff
@@ -68,7 +68,7 @@ export class BidiHTTPRequest extends HTTPRequest {
   }
 
   override resourceType(): ResourceType {
-    return this.initiator().type.toLowerCase() as ResourceType;
```

Comment:
This is totally wrong... We should see if we can add `ResourceType` to the spec. 
Both Firefox and Chrome support similar functionality in there respective DevTools panels.